nixos/modules/services/security/authelia.nix

```
@@ -165,7 +165,7 @@ let
 
             log = {
               level = mkOption {
-                type = types.enum [ "info" "debug" "trace" ];
+                type = types.enum [ "trace" "debug" "info" "warn" "error" ];
                 default = "debug";
                 example = "info";
                 description = "Level of verbosity for logs: info, debug, trace.";
```

Option description is out of sync now and needs updating.

How about:

```suggestion
                description = "Level of verbosity for logs.";
```

---

Yes I think this is the best option, I pushed it